std/eval/_std/mbedtls/Certificate.hx

```diff
@@ -0,0 +1,8 @@
+package mbedtls;
+
+extern class Certificate {
+	function new():Void;
+
+	function parse_file(path:String):Int;
```

Comment:
`@:native` snake case, Haxe camel case?

Reply:
I like to keep externs "clean" and match the native names if at all possible. That's also what I did for `eval.vm.Gc`.